pkg/util/machines_test.go

```diff
@@ -252,12 +252,46 @@ func Test_GetMachineMetadata(t *testing.T) {
 			expected: `
 instance-id: "test-vm"
 local-hostname: "test-vm"
+network:
+  version: 2
+  ethernets:
+    id0:
+      match:
+				macaddress: "00:00:00:00:00"
```

Comment:
```suggestion
        macaddress: "00:00:00:00:00"
```

tabs to spaces will fix the unit tests